docs/stats.md

```diff
@@ -0,0 +1,142 @@
+## accessing stats
+
+Stats can be accessed in two ways:
+
+### control server
+
+`$ pumactl stats` or `GET /stats`
+
+Read more about `pumactl` and the control server [here](https://github.com/puma/puma#controlstatus-server).
```

Comment:
Descriptive link text [for accessibility reasons](https://accessibility.oregonstate.edu/descriptivelinks):

```suggestion
[Read more about `pumactl` and the control server in the README.](https://github.com/puma/puma#controlstatus-server).
```